demo/znkQuestionReport/demo.js

```diff
@@ -0,0 +1,17 @@
+(function(angular) {
+    'use strict';
+    angular.module('demo', [
+        'znk.infra.znkQuestionReport',
+        'demoEnv'
+        ])
+        .controller('Main', function () {
+            'ngInject';
+            var vm = this;
+            vm.questionData = {
+                questionQUID: '123456789',
+                exerciseID: '987654321',
+                exerciseTypeID: '3',
+                questionID: '65498'
+            }
```

Comment:
I think it will be better to use more real data - the numbers here looks too weird